persistence-cassandra-3.11/pom.xml

```diff
@@ -202,20 +202,22 @@
           <instructions>
             <Bundle-Name>Cassandra-Persistence</Bundle-Name>
             <Bundle-Description>Cassandra DB component for project</Bundle-Description>
-            <Bundle-SymbolicName>io.stargate.db.cassandra</Bundle-SymbolicName>
+            <Bundle-SymbolicName>io.stargate.db.cassandra_3_11</Bundle-SymbolicName>
```

Comment:
doesn't it impact our clients? (i.e. CNDB)

Reply:
AFAIK, this ID is internal to OSGi. It does not affect maven dependencies.

I changed it to make it easier to distinguish 3.11 and 4.0 persistence in logs.